scripts/auth-common.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/bash
+
+set -euxo pipefail
```

Comment:
Are you sure you want `-x`, since we're then printing out the password variables too?

Also, no pipes are used in the script, so `-o pipefail` is unnecessary.

Reply:
I don't think we have passwords here now, do we? Only hashes and file paths.

Reply:
Yeah, my bad, I read `HTPASSWD` as `PASSWD`. 

As for the `-o pipefail`, it does no harm here, so I'm fine either way. Just figured it was leftover as there was no pipes.